src/pdffontparsertype1.cpp

```diff
@@ -1212,11 +1211,10 @@ wxPdfFontParserType1::ReadPFM(wxInputStream& pfmFile)
   // the hyphen to a space.  This actually works in a lot of cases.
   wxString fullName = fontName;
   fullName.Replace(wxT("-"), wxT(" "));
-  wxString familyName = wxEmptyString;

```

Comment:
This variable seems completely unused which doesn't look right.

And so is the `fullName` above.